### PR TITLE
Add body field in Rest.request

### PR DESCRIPTION
**Pull request by vloth**
To see the discussion that precedes the pull request: #8 
Happy to change anything you think should be updated!

### CODE REVIEW

**Review by jihchi — CHANGES_REQUESTED** [review]
Nice work! 

Would you mind to move the test with request body to a dedicated test case and leave origin simple test as it is? something like this:

1. `testPromise("post works", () => ...` keep it as a simplest case
1. `testPromise("post with request body", () => ...` the test with request body

**Comment by vloth** [comment]
Hi, sure no problem!
I pushed a commit splitting up the tests for request body from the tests for the regular post.
Could you please take a look?

File: __tests__/MSW_browser.test.js [hunk 1]
```diff
@@ -82,10 +82,25 @@ describe('Service Worker', () => {
...
+      });
+
+      expect(actual).toEqual({ status: 200, body: 'jihchi/res-msw' });
+    })
+
     test('post works', async () => {
       const actual = await page.evaluate(async () => {
         const res = await fetch('https://api.github.com/repos/jihchi/res-msw', {
           method: 'POST',
+          headers: { 'Content-Type': 'application/json' },
```
Comment (jihchi) [inline __tests__/MSW_browser.test.js:102]: **question**: Do we still need this?

File: __tests__/MSW_node_test.res [hunk 2]
```diff
@@ -32,10 +33,33 @@ describe("Node Server", () => {
...
+
     testPromise("post works", () =>
       Fetch.fetchWithInit(
         "https://api.github.com/repos/jihchi/res-msw",
-        Fetch.RequestInit.make(~method_=Post, ()),
+        Fetch.RequestInit.make(
+          ~method_=Post,
+          ~headers=Fetch.HeadersInit.make({"Content-Type": "application/json"}),
+          (),
+        ),
```
Comment (jihchi) [inline __tests__/MSW_node_test.res]: **question**: Do we still need this?

**Review by jihchi — CHANGES_REQUESTED** [review]

File: __tests__/MSW_browser.test.js (hunk 1 above)
Comment (vloth) [reply __tests__/MSW_browser.test.js:102]: No, this can be removed! thanks!

File: __tests__/MSW_node_test.res (hunk 2 above)
Comment (vloth) [reply __tests__/MSW_node_test.res]: I simplified this to just a regular Post, thanks!

**Comment by vloth** [comment]
Hi @jihchi sorry the delay again! I fixed the comments you made, splifiying them! thanks for the review!

**Review by jihchi — APPROVED** [review]

**Comment by jihchi** [comment]
@vloth Thanks very much for contribution! 

The changes has been released in [`v0.6.0`](https://github.com/jihchi/res-msw/releases/tag/v0.6.0)

**Pull request merged**